Node.js 108: Keep App Running (new --watch option) (#2508)
kamranahmedse/developer-roadmap: content/roadmaps/107-nodejs/content/108-nodejs-keep-app-running/readme.md

# Keep App Running

In Node.js, you need to restart the process to make changes take effect. This adds an extra step to your workflow. You can eliminate this extra step by using `nodemon` to restart the process automatically.

> NEW: Now in Node.js 18.11.0, you can run Node with the `--watch` flag to reload your app everytime a file is changed. So you don't need to use `nodemon` anymore.

Reviewer: ```suggestion
Since Node.js 18.11.0, you can run Node with the `--watch` flag to reload your app everytime a file is changed. So you don't need to use `nodemon` anymore.
```
